src/Razor/src/Microsoft.NET.Sdk.Razor/build/netstandard2.0/Microsoft.NET.Sdk.Razor.StaticWebAssets.targets

```diff
@@ -181,6 +187,15 @@ Copyright (c) .NET Foundation. All rights reserved.
     Outputs="$(_GeneratedStaticWebAssetsDevelopmentManifest)"
     DependsOnTargets="$(GenerateStaticWebAssetsManifestDependsOn)">
 
+    <ItemGroup>
+      <_WebRootFiles Include="@(ContentWithTargetPath)" Condition="$([System.String]::Copy('%(TargetPath)').StartsWith('wwwroot'))" />
+      <_ReferencedStaticWebAssets Include="@(StaticWebAsset)" Condition="'%(SourceType)' != ''" />
+    </ItemGroup>
+
+    <ValidateStaticWebAssetsUniquePaths
```

Comment:
Can this be skipped if either of the two item groups is empty?

Reply:
What necessitates this new task? This looks like an additional check we're adding that wasn't present in 3.x?

Reply:
It was an implied check based on the fact that we enforced all base paths to be different and for 1 base path to map to a single content root.

We also didn't support putting assets on the "/" path as we are supporting right now, so there was no chance for the assets to conflict with the ones in the consuming app.

Now that we are relaxing that restriction, we still want to check that in cases where you have multiple content roots associated with the same base path there are no two files that end up in the same path at publish time (as that would mean undefined behavior) and we also want to apply that check against files from the wwwroot folder because now static web assets will be able to be exposed in "/".

To summarize, this restriction was already in place, we are keeping the relaxed version of it.

Does that make sense?